style.css

```diff
@@ -432,14 +441,86 @@
   cursor: not-allowed;
 }
 
-.footer {
-  color: #fff;
+/* The Modal (background) */
+.modal {
+  display: none; /* Hidden by default */
+  position: fixed; /* Stay in place */
+  z-index: 1; /* Sit on top */
+  background-image: url(./assets/ham_bot.svg);
+  background-repeat: no-repeat;
+  background-position: top right;
+  background-size: 30%;
+  left: 0;
+  top: 0;
+  width: 100%; /* Full width */
+  height: 100%; /* Full height */
+  overflow: auto; /* Enable scroll if needed */
+
+  /* Fallback color */
+  background-color: #191675 !important;
+
+  /* Black w/ opacity */
+  background-color: rgba(0, 0, 0, 0.4);
+}
+
+/* Modal Content/Box */
+.modal-content {
+  background-color: rgba(255, 255, 255, 0.16);
+  margin: 51px 24px 151px 24px;
+  padding: 20px;
+  border: 1px solid #fff;
   width: 87.2vw;
-  margin: auto;
+  box-shadow: 0 48px 48px rgba(37, 47, 137, 0.08);
 }
 
-.form {
+/* The Close Button */
+.close {
+  color: #aaa;
+  float: right;
+  font-size: 28px;
+  font-weight: bold;
+}
+
+.close:hover,
+.close:focus {
+  color: black;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+.modal-content .hero2 {
+  display: flex;
+  justify-content: space-between;
+  height: 58px;
+  text-align: center;
+  margin-bottom: 24px;
+}
+
+#img {
+  width: 100%;
+  height: 35%;
+}
+
+.modal-content #img {
+  margin: 24px 24px;
```

Comment:
- [ ] This Margin is moving the image to the right as you can see in your design 👇 
For both the Desktop version 👇 
![Screenshot from 2022-07-29 12-05-47](https://user-images.githubusercontent.com/87965534/181726496-014c4f74-34c4-467c-8f3e-9bfa3561ff71.png)
and the mobile version 👇 
![Screenshot from 2022-07-29 12-06-52](https://user-images.githubusercontent.com/87965534/181726804-234b6b07-4f3b-4115-a19c-29ce87840b22.png)
Please fix it to match the Figma design 👇 
![Screenshot from 2022-07-29 12-05-31](https://user-images.githubusercontent.com/87965534/181727050-835e3f6b-db5c-4a4e-95b2-df24e2acbd37.png)
![Screenshot from 2022-07-29 12-06-34](https://user-images.githubusercontent.com/87965534/181728316-8ece1ad2-eb45-4d5a-9fe7-400ff27b37bf.png)

- [ ] Also the styles of the `See Live` and `See Source` buttons in your project don't match the Figma ones 👇 
Figma design for the desktop version 
![Screenshot from 2022-07-29 12-18-39](https://user-images.githubusercontent.com/87965534/181727912-85b64ae9-0425-4694-bef7-d51e03e39b78.png)
Your design 👇 
![Screenshot from 2022-07-29 12-19-07](https://user-images.githubusercontent.com/87965534/181727996-d5ca01d5-b1a5-4ff7-8744-7bd46a7f1f8a.png)

Figma design for the mobile version 
![Screenshot from 2022-07-29 12-18-50](https://user-images.githubusercontent.com/87965534/181728085-285d3482-eb96-451d-9698-04f66fc5684e.png)
Your design 👇 
![Screenshot from 2022-07-29 12-19-21](https://user-images.githubusercontent.com/87965534/181728131-4fc52250-ad2c-4ee3-8f1d-d4731356c9a2.png)

Reply:
@Sahar-AbdelSamad Consider it done.